charts/prometheus/values.yaml

```diff
@@ -481,39 +477,148 @@ kubeStateMetrics:
 ##
 # kube-state-metrics:
 
-nodeExporter:
+prometheus-node-exporter:
   ## If false, node-exporter will not be installed
   ##
   enabled: true
 
-  ## If true, node-exporter pods share the host network namespace
-  ##
+  # Expose the service to the host network
   hostNetwork: true
 
-  ## If true, node-exporter pods share the host PID namespace
-  ##
+  # Share the host process ID namespace
   hostPID: true
 
-  ## If true, node-exporter pods mounts host / at /host/root
-  ##
-  hostRootfs: true
-
-  ## node-exporter container name
-  ##
-  name: node-exporter
+  # Mount the node's root file system (/) at /host/root in the container
+  hostRootFsMount:
+    enabled: true
+    # Defines how new mounts in existing mounts on the node or in the container
+    # are propagated to the container or node, respectively. Possible values are
+    # None, HostToContainer, and Bidirectional. If this field is omitted, then
+    # None is used. More information on:
+    # https://kubernetes.io/docs/concepts/storage/volumes/#mount-propagation
+    mountPropagation: HostToContainer
 
-  ## node-exporter container image
-  ##
   image:
     repository: quay.io/prometheus/node-exporter
-    tag: v1.3.1
+    # Overrides the image tag whose default is {{ printf "v%s" .Chart.AppVersion }}
+    tag: "v1.3.1"
```

Comment:
not sure if we need to define the tag by default.
will already be updated through chart version.

Reply:
This one is to maintain the compatibility with previous embedded chart.

Reply:
i would still remove it as a chart update of proemtheus would keep the old version of node-exporter otherwise.